cmd/translate/service/service.go

```diff
@@ -78,18 +78,37 @@ var rootCmd = &cobra.Command{
 			log.Panicf("create new repo: %v", err)
 		}
 
-		googleTranslate, closeTranslate, err := fuzzy.NewGoogleTranslate(ctx, fuzzy.WithDefaultClient(ctx))
-		if err != nil {
-			log.Fatalf("create new google translate client: %v\n", err)
+		var (
+			translator    fuzzy.Translator
+			errTranslator error
+		)
+
+		translatorString := viper.GetString("service.translator")
+
+		switch translatorString {
+		case "":
+			translator = &fuzzy.NoopTranslate{}
+		case "AWSTranslate":
+			translator, errTranslator = fuzzy.NewAWSTranslate(ctx, fuzzy.WithDefaultAWSClient(ctx))
+		case "GoogleTranslate":
+			var closeTranslate func() error
+			translator, closeTranslate, errTranslator = fuzzy.NewGoogleTranslate(
+				ctx, fuzzy.WithDefaultGoogleClient(ctx))
+
+			defer func() {
+				if closeErr := closeTranslate(); closeErr != nil {
+					log.Printf("Failed to close GoogleTranslate client: %v\n", closeErr)
+				}
+			}()
+		default:
+			log.Fatalf("unsupported translator: %s\n", translatorString)
```

Comment:
When Fatal is called, deferred functions are not executed. Is this widespread in the Cmd code?

Reply:
I found out that in the cmd package log.Fatal is used in twice.

Reply:
we have a mix of `log.PanicF` and `log.Fatalf`, would be nice to change them to all `log.PanicF`, just for consistency